ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -65,7 +66,13 @@ export class ImportCommand extends IronfishCommand {
   async importFile(path: string): Promise<AccountImport> {
     const resolved = this.sdk.fileSystem.resolve(path)
     const data = await this.sdk.fileSystem.readFile(resolved)
-    return JSONUtils.parse<AccountImport>(data)
+    try {
+      const decodedWords = bech32m.bech32m.decode(data).words
+      const decodedBytes = bech32m.bech32m.fromWords(decodedWords)
+      return JSONUtils.parse<AccountImport>(decodedBytes.toString())
+    } catch (e) {
```

Comment:
Lets not swallow all errors, catch only the JSON parse error type (whatever it is)